src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -626,7 +668,7 @@ namespace DataHeatBalance {
 
         // Default Constructor
         PeopleData()
-            : ZonePtr(0), NumberOfPeople(0.0), NumberOfPeoplePtr(-1), EMSPeopleOn(false), EMSNumberOfPeople(0.0), ActivityLevelPtr(-1),
+            : ZonePtr(0), spaceIndex(0), NumberOfPeople(0.0), NumberOfPeoplePtr(-1), EMSPeopleOn(false), EMSNumberOfPeople(0.0), ActivityLevelPtr(-1),
```

Comment:
Should these spaceIndex vars be initialized to -1? I guess spaces will always exist and this will get set elsewhere so maybe not a big deal.

        EPVector<int> spaceNums;     // Indexes to spaces associated with this input object

Reply:
At the moment `spaceIndex` is used in the same way as `ZonePtr`. `Zone` and `space` are all 1-based (for now), so I think zero is fine but if -1 is preferred now, I can change that.

`EPVector<int> spaceNums;` is in the `GlobalInternalGainMiscObject` which is a temporary holder during input processing. If the internal gain object applies to a single Space, this will hold one number. If it applies to a Zone with more than one space in it, or a ZoneList or SpaceList, then this will hold all of the applicable spaces. If, say  People object is assigned for a Zone that contains 3 Spaces, then it will generate 3 instances of `dataHeatBal->People`, one for each space that it touches (and the gain is apportioned by floor area).

Reply:
I figured you would know better than I how each of these new variables should be initialized so just pointing out things as I go through. You pick and choose which are important for now.